### DISCUSSION TITLE
Some questions about the use of Typography

### DISCUSSION THREAD

**Discussion by gkj17** [material-components/material-web#5554] [Q&A]
### Description

I found that in [Typography](https://github.com/material-components/material-web/blob/main/docs/theming/typography.md).md there seems to be nothing about how to use headline3 or headline4 or ..... subTitle1 subTitle2 as such? So please tell me how I should use it now?

**Comment by asyncliz** [comment]
Material's [applying type guidelines](https://m3.material.io/styles/typography/applying-type) has explanations on the purpose of each typescale role and where to use them.

I created https://github.com/material-components/material-web/pull/5553 to add that info to our typography docs!
